client/state/analytics/actions/record.js

```diff
@@ -42,4 +44,6 @@ export const recordPageView = ( url, title, service, properties = {}, options =
 	},
 } );
 
+export const recordPageView = withEnhancers( recordPageViewBasic, [ enhanceWithSiteMainProduct ] );
```

Comment:
All `recordPageView` will now be enhanced to include the `site_main_product` property, without having to specify them whenever `recordPageView` takes place.

Reply:
Thanks for removing the ambiguity 👍